src/Raven.Server/Documents/Indexes/Static/MapIndexDefinition.cs

```diff
@@ -122,9 +123,8 @@ public static IndexDefinition Load(StorageEnvironment environment, out long vers
                     definition.Name = ReadName(reader);
                     definition.LockMode = ReadLockMode(reader);
                     definition.Priority = ReadPriority(reader);
-
                     version = ReadVersion(reader);
-
+                    definition.State = version < 50000 ? IndexState.Normal : ReadState(reader);
```

Comment:
Do we need to bump the index version?
@ppekrol does index rebuild is needed to update the version of an existing index?